src/java/org/apache/cassandra/streaming/SessionInfo.java

```diff
@@ -149,6 +149,14 @@ public long getTotalSizeToReceive()
         return getTotalSizes(receivingSummaries);
     }
 
+    public long getTotalBytesToReceive()
```

Comment:
Can you add a java doc and put in the doc why this method != `getTotalSizeToReceive`?